README.md

```diff
@@ -118,7 +119,33 @@ To configure your documentation to use exapandable sidebar headings like the exa
 
 *Tip: if you want to add all files in a sub-directory to your expandable dropdown section you can use the `:glob:` directive instead of listing out each page (see example above for the How-to Guides, API Reference and Explanations sections). This will list out each page in alphabetical order, so if you want a specific order you will need to list the pages out individually in the `toctree` (see example above for the Tutorials section)*
 
+## Enable translations
+
+First, coordinate with the Translations team at https://github.com/qiskit-community/qiskit-translations to express your interest and to coordinate setting up the infrastructure.
+
+Once the Translations team is ready, then update your `conf.py`:
+
+* Ensure that `qiskit_sphinx_theme` is in the `extensions` setting.
```

Comment:
Sort of. If you don't add `qiskit_sphinx_theme` to extensions, then you need to add `sphinxcontrib.jquery` to the `extensions` list because Pytorch requires jQuery. At least until we switch to Furo.

So, I think we can simplify things to say that you should _always_ add `qiskit_sphinx_theme` to the extensions list.